utility_analysis_new/combiners.py

```diff
@@ -550,6 +552,105 @@ def compute_metrics(self,
                         acc: AccumulatorType) -> metrics.AggregateErrorMetrics:
         """Computes metrics based on the accumulator properties."""
         return metrics.AggregateErrorMetrics(
+            metric_type=metrics.AggregateMetricType.COUNT,
+            abs_error_expected=acc.abs_error_expected /
+            acc.kept_partitions_expected,
+            abs_error_variance=acc.abs_error_variance /
+            acc.kept_partitions_expected,
+            abs_error_quantiles=[
+                sum / acc.kept_partitions_expected
+                for sum in acc.abs_error_quantiles
+            ],
+            rel_error_expected=acc.rel_error_expected /
+            acc.kept_partitions_expected,
+            rel_error_variance=acc.rel_error_variance /
+            acc.kept_partitions_expected,
+            rel_error_quantiles=[
+                sum / acc.kept_partitions_expected
+                for sum in acc.rel_error_quantiles
+            ])
+
+    def metrics_names(self) -> List[str]:
+        return [
+            'abs_error_expected', 'abs_error_variance', 'abs_error_quantiles',
+            'rel_error_expected', 'rel_error_variance', 'rel_error_quantiles'
+        ]
+
+    def explain_computation(self):
+        pass
+
+
+class PrivacyIdCountAggregateErrorMetricsCombiner(pipeline_dp.Combiner):
```

Comment:
What do you think about the following unifictation of count and privacy id count cross-partition metrics:
1. To Drop `PrivacyIdCountMetrics` and to use `CountMetrics`. It looks like the only different between them is that CountMetrics has   `per_partition_error` which should be set `0` for `PRIVACY_ID_COUNT`
2. To use `CountAggregateErrorMetricsCombiner` for both `COUNT` `PRIVACY_ID_COUNT` cross-partition aggregations.